src/charts/LineChart.js

```diff
@@ -20,29 +20,25 @@ function sortByDateAscending( a, b ) {
 }
 
 function findBestYTickFactor( ymin, ymax, factor ) {
-  var multipliers = [ 2, 4, 5, 10, 20, 25, 50, 100, 200, 500, 1000, 2000, 5000, 10000 ];
-  var divisors = [ .5, .25, .1, .05, .025, .01 ];
-  var count = ( ymax - ymin ) / factor;
+  var multipliers = [ 2, 4, 5, 10, 15, 20, 25, 50, 100, 200, 500, 1000, 2000, 5000, 10000 ];
+  var divisors = [ .5, .25, .2, .1, .05, .025, .01 ];
+  var count = Math.ceil( ( ymax - ymin ) / factor );
   var newFactor = factor;
```

Comment:
this is just a code style thing, but it would be helpful to leave this null so that the code errors if the newFactor value doesn't get updated as expected